data-share-standalone.properties

```diff
@@ -0,0 +1,15 @@
+# Enables the data-share application in standalone mode.
+mosip.data.share.standalone.mode.enabled=true
+# Defines the policy json which will be taken into consideration if
+# "mosip.data.share.standalone.mode.enabled" is set as true.
+# If we are using "encryptionType" as "Partner based" then subscriberId must be a valid subscriberId
+# i.e. should exist in system.
+mosip.data.share.static-policy.policy-json={"typeOfShare":"","transactionsAllowed":"2","shareDomain":"datashare.datashare","encryptionType":"NONE","source":"","validForInMinutes":"30"}
+# Defines the policyId which will be taken into consideration if "
+# mosip.data.share.standalone.mode.enabled" is set as true.
+mosip.data.share.static-policy.policy-id=static-policyid
+# Defines the subscriberId which will be taken into consideration if
+# "mosip.data.share.standalone.mode.enabled" is set as true.
+mosip.data.share.static-policy.subscriber-id=static-subscriberid
+# Disables JWT signature computation while storing object in object store.
+mosip.data.share.signature.disabled=true
```

Comment:
We will also need the data-share-standalone.properties here.

Reply:
Done the changes.